addons/website_sale/views/product_image_views.xml

```diff
@@ -44,18 +44,20 @@
                 <field name="id"/>
                 <field name="name"/>
                 <field name="image_1920"/>
+                <field name="image_128"/>
+                <field name="video_url"/>
                 <field name="sequence" widget="handle"/>
                 <templates>
                     <t t-name="kanban-box">
-                        <div class="card oe_kanban_global_click p-0">
+                        <div class="card p-0">
                             <div class="o_squared_image">
-                                <img class="card-img-top" t-att-src="kanban_image('product.image', 'image_1920', record.id.raw_value)" t-att-alt="record.name.value"/>
+                                <field class="card-img-top" name="image_1920" widget="website_sale_image_field"/>
```

Comment:
is working in image_128 or other image sizes? 

please add a test case.

Reply:
Test case will be added soon. I have adapted the code as per our supported image fields, and dynamically assigned the data.